bin/heudiconv

```
@@ -657,26 +668,36 @@ def convert_dicoms(subjs, dicom_dir_template, outdir, heuristic_file, converter,
                 "atm we support processing only 1 subject/session at a time"
             sess, sid = session_subject_ids.pop()
 
+
+        if is_bids and not sid.isalnum(): # alphanumeric only
+            old_sid = sid
```

i would just raise an error and have the user respecify the sid